src/Microsoft.Identity.Web.AgentIdentities/Microsoft.Identity.Web.AgentIdentities.csproj

```diff
@@ -8,8 +8,7 @@
 
     <!-- The package is new in 3.10.0.-->
     <PackageValidationBaselineVersion>3.10.0</PackageValidationBaselineVersion>
-    <EnablePackageValidation>false</EnablePackageValidation>
-    
+    <EnablePackageValidation>true</EnablePackageValidation>
```

Comment:
now that we released a first version of Microsoft.Identity.Web.AgentIdentities, we can enable package (version) validation